tests/test_flow/test_persistence_random.py

```diff
@@ -179,8 +237,15 @@ class ModelEntity:
     binding = attr.ib(default=None)
     dependent_entities = attr.ib(default=attr.Factory(list))
 
-    def compute_value(self, dep_values):
-        return f"{self.name}.{self.binding.func_version}({','.join(dep_values)})"
+    def compute_value(self, func_version, dep_values):
```

Comment:
nit: Do you need to pass the func version here? I think it was cleaner when it was reading from binding directly.

Reply:
Nope, there's no good reason for this. This is left over from a different approach. Thanks!